app/src/main/java/com/example/practicals/kotlin_practicals/ExceptionTest.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.practicals.kotlin_practicals
+
+class ExceptionTest {
+
```

Comment:
if class is not necessary then remove it. if you need it then put comment inside that why you need it.